PCLConfig.cmake.in

```diff
@@ -99,23 +99,7 @@ macro(find_boost)
     set(Boost_USE_STATIC @Boost_USE_STATIC@)
     set(Boost_USE_MULTITHREAD @Boost_USE_MULTITHREAD@)
   endif()
-  set(Boost_ADDITIONAL_VERSIONS
-    "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@.@Boost_SUBMINOR_VERSION@" "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@"
-    "1.73.0" "1.73" "1.72.0" "1.72" "1.71.0" "1.71" "1.70.0" "1.70"
-    "1.69.0" "1.69" "1.68.0" "1.68" "1.67.0" "1.67" "1.66.0" "1.66" "1.65.1" "1.65.0" "1.65"
-    "1.64.0" "1.64" "1.63.0" "1.63" "1.62.0" "1.62" "1.61.0" "1.61" "1.60.0" "1.60"
-    "1.59.0" "1.59" "1.58.0" "1.58" "1.57.0" "1.57" "1.56.0" "1.56" "1.55.0" "1.55")
-  # Disable the config mode of find_package(Boost)
-  set(Boost_NO_BOOST_CMAKE ON)
   find_package(Boost 1.55.0 ${QUIET_} COMPONENTS @PCLCONFIG_AVAILABLE_BOOST_MODULES@)
-
-  set(BOOST_FOUND ${Boost_FOUND})
```

Comment:
Would this stop overriding the Boost_LIBRARIES variable for anyone consuming PCL? There's an issue open (and verified) which says that `find_package(Boost)` before `find_package(PCL)` causes PCL to override the Boost libraries

Reply:
Also, what's the relationship of `pcl_find_boost.cmake` with `Modules/FindBoost.cmake` (because I notice it's untouched in this PR)

Reply:
> Also, what's the relationship of `pcl_find_boost.cmake` with `Modules/FindBoost.cmake` (because I notice it's untouched in this PR)
That is to be removed.

> Would this stop overriding the Boost_LIBRARIES variable for anyone consuming PCL? There's an issue open (and verified) which says that `find_package(Boost)` before `find_package(PCL)` causes PCL to override the Boost libraries

Currently, no. It should be possible though to append the already existing found components to the list of found components.